docs/user-guide.md

```diff
@@ -46,7 +46,7 @@ To get started using Akri, you must first decide what you want to discover and w
         ```
     1. If using **Kubernetes**, Helm and crictl do not require additional configuration.
 
-1. Install Akri Helm chart and enable the desired Configuration (in this case, ONVIF is enabled). See the [ONVIF Configuration documentation](./onvif-sample.md) to learn how to customize the Configuration. Instructions on deploying the udev Configuration can be found in [this document](./udev-sample.md).
+1. Install Akri Helm chart and enable the desired Configuration (in this case, ONVIF is enabled). See the [ONVIF Configuration documentation](./onvif-sample.md) to learn how to customize the Configuration. Instructions on deploying the udev Configuration can be found in [this document](./udev-sample.md). To allow the controller to run on a node other than the control plane, add `--set controller.onlyOnMaster=false` to your install command below. 
```

Comment:
Should onlyOnMaster be false by default?

Reply:
is there a reason to ever have it be only on master?  i don't know why i set it at first ... if we never had, would we adding a way to enforce that now?

Reply:
You could argue that controllers might make sense to be collocated and closer to etcd/apiservers. Re adding the enforcement now, it probably would not be on my high priority list.

Reply:
Also, if you have dedicated control plane nodes, it makes sense to have controller there instead of using resources and be possibly exposed to bad players on worker nodes.

Reply:
seems like consensus is to leave it to true by default? I see both sides here

Reply:
I believe both me and Brian are advocating for switching to false. But Brian, please correct me if I misrepresented you.

Reply:
i agree with it being false.  i'm not clear, at this point, why it should ever be true.